packages/frontend/src/components/form/ImageFileInput.tsx

```diff
@@ -15,9 +16,9 @@ const ImageFileInput = (name: string, src: string | undefined): JSX.Element => {
 
           <div className="mt-2">
             {src ? (
-              <img src={src} className="w-auto h-48 block" />
+              <img src={imageFullPath(src)} className="block w-auto h-48" />
```

Comment:
Same note as above with `getGenericSettingValueRealized` -- I think the previous architecture is preferable (where the image's full url is defined by the server).

Reply:
Fixed